Refactoring and adding Remote collaboration chapter from old PRs (#1567)
alan-turing-institute/the-turing-way: book/website/collaboration/remote-collab/remote-collab-prosandcons.md

(cl-remote-collab-prosandcons)=
# Perceived Pros and Cons

*Light opinions on people's experience with remote collaboration*

## Pros

- Remote collaboration dramatically improves the life quality of those who cannot comfortably travel or afford to live close to their office.

>   - It has a positive impact on the environment by reducing travel related co2 emissions

Reviewer: ```suggestion
  - It has a positive impact on the environment by reducing travel related CO2 emissions
```

> - A well established format work well equally for low resource environment as the overall organisational cost can be reduced by not having to deal with on site expense

Reviewer: ```suggestion
- A well established format works equally well for low resource environments as the overall organisational cost can be reduced by not having to deal with on site expense
```

> - Distributed team format attracts people from different time zones, different workplace requirements or those who work best with flexible working hour arrangement.
> - This can lead to innovative way to establish social interactions and social dynamics in general
> - It will help us learn about more colleagues from different parts of the world and make us more educated about our cultural differences and how to accommodate in our work environment

Reviewer: ```suggestion
- Distributed team format attracts people from different time zones, different workplace requirements or those who work best with flexible working hour arrangements
- This can lead to innovative ways to establish social interactions and social dynamics in general
- It will help us learn about more colleagues from different parts of the world and educate us more about our cultural differences and accommodations to make in our work environment
```

> - It will make it convenient to have decision making efficiently by involving every stakeholders irrespective of their locations, given the meeting is done in a mutually convenient time

Reviewer: ```suggestion
- Decision making will become more efficient by involving all stakeholders irrespective of their locations, given the meeting is done at a mutually convenient time
```

> As someone who used to commute 4 hours per day for 12 years, I can sum my experiences in two words: remote working reduces inequality. Pablo Rodríguez-Sánchez (GitHub: PabRod)

## Cons

> - Developing camaraderie might be a bit challenging - as we are used to developing those within in-person relationships.
> - It would be hard to separating working hours from the time that should be used for personal life and relationships.
> - If running an online meeting, allocated time might not be enough for arriving to a conclusion and may sometime lead to misinterpretation of decisions made by the team.
> - Conflict resolution maybe difficult if most interactions happen in an allocated time slot.
> - Sometimes such interactions may lack body language, or tone (online chat) and may lead to misunderstanding or unnecessary conflict.
> - May lose the possibility of chanced interaction or water-cooler-effects and innovative ideas that they results.
> - People will have different working condition, for example, limited access to hardware, that can lead to wrong assumptions regarding replication of certain task
> - May lead to distractions if working from a busy co-working space or home with not sufficient arrangement for work that can make it difficult for some people to focus and have disciplined work practices.

Reviewer: ```suggestion
- Developing camaraderie may be challenging as we are used to developing this in-person
- The boundaries between working life and personal life become more blurred
- If running an online meeting, the allocated time might not be enough to arrive at a conclusion and may sometime lead to misinterpretation of decisions made by the team
- Conflict resolution maybe difficult if most interactions happen in an allocated time slot
- Sometimes such interactions may lack body language or tone (online chat) and may lead to misunderstanding or unnecessary conflict
- May lose the possibility of chanced interaction or water-cooler-effects and innovative ideas that they produce
- People will have different working conditions, for example, limited access to hardware that can lead to wrong assumptions regarding replication of certain task
- May be distracting if working from a busy co-working space or home without a sufficient arrangement for work that can make it difficult for some people to focus and have disciplined work practices
```
